src/iterative_sorting/iterative_sorting.py

```diff
@@ -22,14 +21,22 @@ def selection_sort(arr):
 
 # TO-DO:  implement the Bubble Sort function below
 def bubble_sort(arr):
+    # initializes sorted to False so loop can start
```

Comment:
Yay comments! You can use this repo in the future if any interviews ask you about sorting algos.